ORST/Assets/Scripts/Core/Interactions/DoorZone/DoorZone.cs

```diff
@@ -10,7 +11,9 @@
 
 namespace ORST.Core.Interactions {
     public class DoorZone : MonoBehaviour {
+        public Action ExitedDoor;
```

Comment:
Does this need to be invoked from other scripts? If not, let's make it an `event`. If yes, we should expose that functionality with a method and not directly on the `Action`.

```suggestion
        public event Action ExitedDoor;
```